ci/workflows/main-cron.yml

```diff
@@ -581,3 +581,66 @@ steps:
           - "17"
     timeout_in_minutes: 10
     retry: *auto-retry
+
+  - label: "release"
+    command: "ci/scripts/release.sh"
+    if: build.tag != null
+    plugins:
+      - seek-oss/aws-sm#v2.3.1:
+          env:
+            GITHUB_TOKEN: github-token
+      - docker-compose#v4.9.0:
+          run: release-env
+          config: ci/docker-compose.yml
+          mount-buildkite-agent: true
+          environment:
+            - GITHUB_TOKEN
+            - BUILDKITE_TAG
+            - BUILDKITE_SOURCE
+    timeout_in_minutes: 60
+    retry: *auto-retry
+
+  - label: "release docker image: amd64"
+    command: "ci/scripts/docker.sh"
+    key: "build-amd64"
+    if: build.tag != null
+    plugins:
+      - seek-oss/aws-sm#v2.3.1:
+          env:
+            GHCR_USERNAME: ghcr-username
+            GHCR_TOKEN: ghcr-token
+            DOCKER_TOKEN: docker-token
+            GITHUB_TOKEN: github-token
+    timeout_in_minutes: 60
+    retry: *auto-retry
+
+  - label: "docker-build-push: aarch64"
+    command: "ci/scripts/docker.sh"
+    key: "build-aarch64"
+    if: build.tag != null
+    plugins:
+      - seek-oss/aws-sm#v2.3.1:
+          env:
+            GHCR_USERNAME: ghcr-username
+            GHCR_TOKEN: ghcr-token
+            DOCKER_TOKEN: docker-token
+            GITHUB_TOKEN: github-token
+    timeout_in_minutes: 60
+    agents:
+      queue: "linux-arm64"
+    retry: *auto-retry
+
+  - label: "multi arch image create push"
+    command: "ci/scripts/multi-arch-docker.sh"
+    if: build.tag != null
+    depends_on:
+      - "build-amd64"
+      - "build-aarch64"
+    plugins:
+      - seek-oss/aws-sm#v2.3.1:
+          env:
+            GHCR_USERNAME: ghcr-username
+            GHCR_TOKEN: ghcr-token
+            DOCKER_TOKEN: docker-token
+    timeout_in_minutes: 10
+    retry: *auto-retry
```

Comment:
Should we run all of these daily?

Reply:
This jobs will trigger only build has tag.
```
if: build.tag != null
```